lib/muffin_man/notifications/v1.rb

```diff
@@ -0,0 +1,63 @@
+module MuffinMan
+  module Notifications
+    require "json"
+    require "sp_api_helpers"
+    class V1 < SpApiClient
+
+      NOTIFICATION_PATH = "/notifications/v1"
+
+      def create_destinations(arn, params={})
```

Comment:
Same comment here as above
```suggestion
      def create_destination(arn, params={})
```